example-plugins/charting_api_examples/README.md

```diff
@@ -0,0 +1,216 @@
+
+#### Example Response
+
+```json
+{
+  "next_page": "https://training.canvasmedical.com/plugin-io/api/charting_api_examples/notes/?limit=2&offset=6",
+  "count": 2,
+  "notes": [
+    {
+      "id": "10ff2047-6301-4ab4-81cd-b500e7df8ef7",
+      "patient_id": "5350cd20de8a470aa570a852859ac87e",
+      "provider_id": "5843991a8c934118ab4f424c839b340f",
+      "datetime_of_service": "2025-02-21 23:31:45.627894+00:00",
+      "note_type": {
+        "id": "c5df4f03-58e4-442b-ad6c-0d3dadc6b726",
+        "name": "Office visit",
+        "coding": {
+          "display": "Office Visit",
+          "code": "308335008",
+          "system": "http://snomed.info/sct"
+        }
+      }
+    },
+    {
+      "id": "4dba128f-96cc-4dd0-814b-a064bfdcde7e",
+      "patient_id": "5350cd20de8a470aa570a852859ac87e",
+      "provider_id": "336159560091471cb6b0e149d9054697",
+      "datetime_of_service": "2025-02-21 23:31:45.928071+00:00",
+      "note_type": {
+        "id": "c5df4f03-58e4-442b-ad6c-0d3dadc6b726",
+        "name": "Office visit",
+        "coding": {
+          "display": "Office Visit",
+          "code": "308335008",
+          "system": "http://snomed.info/sct"
+        }
+      }
+    }
+  ]
+}
+```
+
+### Read a Note
+
+`GET /plugin-io/api/charting_api_examples/notes/<note-id>/`
+
+#### Example Response
+
+```json
+{
+  "note": {
+    "id": "1490b8db-00a9-47d9-9170-ec142460b586",
+    "patient_id": "5350cd20de8a470aa570a852859ac87e",
+    "provider_id": "6b33e69474234f299a56d480b03476d3",
+    "datetime_of_service": "2025-10-02 23:30:00+00:00",
+    "state": "NEW",
+    "note_type": {
+      "id": "c5df4f03-58e4-442b-ad6c-0d3dadc6b726",
+      "name": "Office visit",
+      "coding": {
+        "display": "Office Visit",
+        "code": "308335008",
+        "system": "http://snomed.info/sct"
+      }
+    }
+  }
+}
+```
+
+### Create a Note
+
+`POST /plugin-io/api/charting_api_examples/notes/`
+
+#### Example Request Body
+
+```json
+{
+  "practice_location_id": "306b19f0-231a-4cd4-ad2d-a55c885fd9f8",
+  "note_type_id": "c5df4f03-58e4-442b-ad6c-0d3dadc6b726",
+  "patient_id": "5350cd20de8a470aa570a852859ac87e",
+  "provider_id": "6b33e69474234f299a56d480b03476d3",
+  "datetime_of_service": "2025-10-04 23:30:00",
+  "title": "My cool note"
+}
+```
+
+### Add Billing Line Item to a Note
+
+`POST /plugin-io/api/charting_api_examples/notes/<note-id>/billing_line_items/`
+
+#### Example Request Body
+
+```json
+{
+  "cpt_code": "98008"
+}
+```
+
+### Add a Diagnose Command to a Note
+
+`POST /plugin-io/api/charting_api_examples/notes/<note-id>/diagnose/`
+
+`icd10_code` is required, but `committed` may be true, false, or omitted entirely.
+
+#### Example Request Body
+
+```json
+{
+  "icd10_code": "E119",
+  "committed": true
+}
+```
+
+### Add Multiple Commands to a Note
+
+`POST /plugin-io/api/charting_api_examples/notes/<note-id>/prechart/`
+
+#### Example Request Body
+
+```json
+null
```

Comment:
Is this valid json? I would expect it to be `{}`

Reply:
It is! @csande recently educated me on this and I tested it!